Event Delegation/app.js

```diff
@@ -0,0 +1,9 @@
+let inputItems = document.querySelectorAll('input');
+
```

Comment:
Please use `const` since the variable value was not reassigned.

Reply:
Thanks, @lycoangelo, This is now updated.